### ISSUE TITLE
Feedback on alt text flow

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode-copilot-vision#86]
From @rperez030, cc @jooyoungseo for thoughts:

"if we really want this tool to help people create good alt text, it probably should ask what is the purpose of the image with a list of predefined options (E.G informative, descriptive, functional, branding...) and generate the alt text based on the selection. For example, if the image in the video is intended to be used as a link to the home page, that wouldn't be a good alt text.?"

While I agree with this feedback, my concern is that a quick fix or code action is intended to be applied immediately without additional input/feedback. I wonder if we should have another action, something like "generate alt text with intent" to allow for this as an option for users?

**Comment by meganrogge** [comment]
also cc @Tyriar and @justschen for thoughts

**Comment by justschen** [comment]
@meganrogge not sure if there is much we can do with the extension, but i reckon once this is actually in copilot, it could be a custom code action in these scenarios that could open inline chat, where we can take advantage of applying/discarding edits and query history

**Comment by meganrogge** [comment]
I think in the extension, we could just add a step to the quick pick that allows users to choose one of these. I like the idea of waiting on this until it's in copilot though.
